### PR TITLE
[13.0][MIG] maintenance_stock: Migration to 13.0

### PR DESCRIPTION
**Pull request by mamcode**

### CODE REVIEW

**Comment by mamcode** [comment]
@dalonsod Please can you help me with a review ?

**Comment by mamcode** [comment]
@rhe-mastercore maybe you can help with a review here :)

File: maintenance_stock/models/stock_warehouse.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                "name": self.name + " " + _("Sequence consumption"),
+                "prefix": self.code + "/CONS/",
+                "padding": 5,
+                "company_id": self.company_id.id,
+            },
+        }
+
+    def _get_locations_values(self, vals, code=False):
+        sub_locations = super()._get_locations_values(vals)
+        code = vals.get("code") or self.code
```
Comment (dalonsod) [inline maintenance_stock/models/stock_warehouse.py]: I'm not sure, but with new `_get_locations_values` definition, should this code be like this (not tested)?
```suggestion
    def _get_locations_values(self, vals, code=False):
        sub_locations = super()._get_locations_values(vals, code=code)
        code = vals.get("code") or code or self.code
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Thanks for the contribution! Only code review

File: maintenance_stock/models/stock_warehouse.py (hunk 1 above)
Comment (mamcode) [reply maintenance_stock/models/stock_warehouse.py]: you're right

**Comment by mamcode** [comment]
The Job pre-commit is falling but I don't know how to resolve it.
It seems to have nothing to do with the code of this PR
@dalonsod Can you help me ?

**Comment by dalonsod** [comment]
I'm sorry, I don't know why that error is raised. Anyway, it could be only a temporary issue, could you update PR with a dummy commit, so github actions should be fired again and check if error persists?

**Comment by mamcode** [comment]
@dalonsod Review again please.

**Review by dalonsod — APPROVED** [review]
Tested 👍  using existing and new warehouses.

**Review by etobella — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**